README.md

```diff
@@ -13,23 +13,23 @@
 > [!NOTE]  
 > Wait, wasn't this project called Kinobi before? We've [renamed this project to **Codama**](https://github.com/codama-idl/codama/pull/234).
 
-Codama is a tool that describes any Solana program in a standardised format called a **Codama IDL**. 
+Codama is a tool that describes any Solana program in a standardised format called a **Codama IDL**.
```

Comment:
These README changes are from `lint:fix` I think, shall I revert?

Reply:
No that's fine, it's better like that. ☺️